Python 3 support and prompt for user/password when missing from .pepperrc

buchanan opened the pull request.

whiteinge commented:
Great additions!

I'm seeing the following traceback after inputting a username at the prompt:

```python
Traceback (most recent call last):
  File "/Users/shouse/src/salt/pepper/scripts/pepper", line 26, in <module>
    exit_code, results = cli.run()
  File "/Users/shouse/src/salt/pepper/pepper/cli.py", line 338, in run
    creds = iter(self.parse_login())
  File "/Users/shouse/src/salt/pepper/pepper/cli.py", line 253, in parse_login
    login_details = self.get_login_details()
  File "/Users/shouse/src/salt/pepper/pepper/cli.py", line 232, in get_login_details
    results['SALTAPI_USER'] = input('Username: ')
  File "<string>", line 1, in <module>
NameError: name 'saltdev' is not defined
```

whiteinge commented on pepper/libpepper.py:
```diff
@@ -7,9 +7,13 @@
 import os
 try:
+    import ssl
+    ssl._create_default_https_context = ssl._create_unverified_context
```
Do you know what Python versions this assignment will work for? I'm under the impression it works for 2.6 but I'm not sure about 2.5.

buchanan commented:
I've fixed the error you were experiencing when entering a username.
I also changed the default ssl context to ssl._create_stdlib_context which I believe will be more universal with older versions of python however I ran into several issues when trying to use pepper in python <2.6

whiteinge commented:
Thanks for the additions! And thanks for the heads-up about 2.6. I'll take a closer look there.
